pir/cpp/payload.h

```diff
@@ -30,40 +31,99 @@ using seal::GaloisKeys;
 using std::optional;
 using buff_type = std::vector<seal::Ciphertext>;
 
-class PIRPayload {
+class PIRCiphertexts {
```

Comment:
These classes seem to largely duplicate the functionality of the protocol buffers. How about just a utility function to extract the ciphertexts, rather than defining a set of mirror classes?